.flowconfig

```diff
@@ -37,24 +34,22 @@ emoji=true
 esproposal.optional_chaining=enable
 esproposal.nullish_coalescing=enable
 
+exact_by_default=true
```

Comment:
that's how we do it. by default, also exact. :D

but for curiosity: what does it mean?

Reply:
Honestly, I'm not entirely sure, I just followed the upgrade docs. flow was not on a priority list as it's never blocking releases anyway. I've put a lot of `$FlowFixMe` comments in the code just to get CI to pass, I suggest we leave resolving these for a separate PR, once we're back on track.